package.json

```
@@ -44,8 +44,8 @@
     "lint": "run-s lint:*",
     "lint:js": "npm run lint-js .",
     "lint-js": "eslint --cache --cache-location .cache/.eslintcache",
-    "lint:markdown": "npm run lint-markdown *.md",
-    "lint-markdown": "markdownlint --config ./.markdownlint.json --ignore '.vale/**/*.md' --ignore '.github/**/*.md'",
+    "lint:markdown": "npm run lint-markdown '**/*.{md,mdx}'",
```

```suggestion
    "lint:markdown": "npm run lint-markdown '*.{md,mdx}'",
```

previously it was `*.md` and now it is `**/*` this will miss the readme.md in the main directory? and run only for sub directories? correct me if i'm wrong?

---

I believe the one before was wrong since `*.md` won't match markdown files recursively. `markdownlint-cli` uses `glob` for those patterns, as you can see from https://github.com/isaacs/node-glob/blob/main/README.md:

> ** If a "globstar" is alone in a path portion, then it matches zero or more directories and subdirectories searching for matches. It does not crawl symlinked directories.

So `readme.md` will be included as well.

---

✅ LGTM